homeassistant/components/statistics/sensor.py

```diff
@@ -638,7 +645,7 @@ def _async_scheduled_update(self, now: datetime) -> None:
             self.async_write_ha_state()
 
     def _fetch_states_from_database(self) -> list[State]:
-        """Fetch the states from the database."""
+        """Fetch the states from the database. In principle this should also load the last value before the time interval for the time based functions. However, this would mean loading the complete history without time or number limit, which is maybe a little too much effort."""
```

Comment:
Lines should not be longer than 88 characters.

Reply:
Ok, done. Shouldn't that be checked by one of the pre-commit hooks?